internal/commands/suggest_builders.go

```diff
@@ -20,18 +20,18 @@ type suggestedBuilder struct {
 
 var suggestedBuilders = []suggestedBuilder{
 	{
-		Name:               "Cloud Foundry",
-		Image:              "cloudfoundry/cnb:bionic",
-		DefaultDescription: "Small base image with Java & Node.js",
+		Name:               "Paketo Buildpacks",
```

Comment:
Hi @kvedurmu, 

This isn't detailed anywhere that I can remember but there is an implicit alphabetical ordering that we are trying to adhere to here. That would mean that these should now be _after_ the heroku builders.

Reply:
Hey @jromero It looks like there's some sorting logic [here](https://github.com/buildpacks/pack/blob/master/internal/commands/suggest_builders.go#L67) that sorts by the `Image` path instead of the `Name`. So according to that the sorting here is correct, but perhaps it makes sense to sort by `Name` instead?

Reply:
Good point. We'll accept this as is and make a few tweaks as we deem necessary.